superset/security.py

```diff
@@ -133,6 +139,39 @@ def _search_ldap(self, ldap, con, username):
             if not user[0][0]:
                 return None
         return user
+    
+    def decryptMessage(self, message):
+        if message :
+            _e = base64.b64decode(message)
+            _i = _e[:BS]
+            _a = AES.new(SK, AES.MODE_CBC, _i)
+            _d = _a.decrypt(_e[BS:]).decode('utf-8')
```

Comment:
we should some better variable names.

Reply:
because of security ,using short named variable